examples/getting_started/README.md

```diff
@@ -0,0 +1,29 @@
+# Getting Started with NVFlare
+NVFlare is an open-source framework that allows researchers and data scientists to seamlessly move their 
+machine learning and deep learning workflows into a federated paradigm.
```

Comment:
This PR is good, I have one question of when users come in, do we point them to hello-world or getting_started? 

Do we need to consolidate these two?

Reply:
Getting started. This is what is the entry point on the website.